larray/inout/csv.py

```diff
@@ -118,26 +118,32 @@ def read_csv(filepath_or_buffer, nb_axes=None, index_col=None, sep=',', headerse
     Specify the number of axes of the output array (useful when the name of the last axis is implicit)
 
     >>> fname = os.path.join(EXAMPLE_FILES_DIR, 'missing_axis_name.csv')
-    >>> # let's take a look inside the CSV file.
-    >>> # The name of the second axis is missing.
+    >>> # let's take a look inside the CSV file 'missing_axis_name.csv'.
+    >>> # The name of the last axis is missing.
     >>> with open(fname) as f:
     ...     print(f.read().strip())
-    a,b0,b1,b2
-    a0,0,1,2
-    a1,3,4,5
-    a2,6,7,8
-    >>> # read the array stored in the CSV file as is
-    >>> read_csv(fname)
-    a\{1}  b0  b1  b2
-       a0   0   1   2
-       a1   3   4   5
-       a2   6   7   8
+    a,b,c0,c1
+    a0,b0,0,1
+    a0,b1,2,3
+    a1,b0,4,5
+    a1,b1,6,7
+    >>> # read the array stored in the CSV file as it
+    >>> arr = read_csv(fname)
+    >>> arr.info
+    4 x 3
+     a [4]: 'a0' 'a0' 'a1' 'a1'
+     {1} [3]: 'b' 'c0' 'c1'
+    dtype: object
+    memory used: 96 bytes
```

Comment:
adding a comment somewhere to say that this is not what we want/need/expected would help I think